.travis.yml

```diff
@@ -18,6 +18,7 @@ before_install:
 
 install:
   - ./travis-tool.sh install_r RJSONIO
+  - ./travis-tool.sh install_r curl
```

Comment:
If both go to `install_r` you can just have multiple args on one line.

But it is faster to do `install_aptget r-cran-curl` which should (!!) be resolvable.  Else I address.